server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -394,6 +398,20 @@ private static long jitter() {
         * TabletServer.TIME_BETWEEN_LOCATOR_CACHE_CLEARS);
   }
 
+  public Semaphore getSemaphore() {
```

Comment:
Was wondering how to make this cheaper when the semaphore was not needed.  Thought one way was to use optional.

```suggestion
  public Optional<Semaphore> getSemaphore() {
```